### PR TITLE
Bookstore

### PR DESCRIPTION
**Pull request by 29td**
Created a react app
Setup routing using react router.

### CODE REVIEW

File: src/Components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const Navbar = () => (
+  <header>
+    <nav>
+      <ul>
+        <li>
+          <NavLink to="/">Books</NavLink>
+        </li>
+        <li>
+          <NavLink to="/">Categories</NavLink>
```
Comment (TedLivist) [inline src/Components/Navbar.js]: - [x] Navigation to the categories page does not really work. To match the routing in the App component, this should be:

  ```
  ...
  <NavLink to="/categories">Categories</NavLink>
  ...
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @29td,

Good job so far!
✔️ Use of reusable components
✔️ Modularization of code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly format your entire README using [the Microverse template](https://github.com/microverseinc/readme-template)

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @29td,

✔️ Good documentation
✔️ Use of RRD for navigation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
Thank you 👍

**Pull request merged**